nav2_collision_monitor/src/costmap.cpp

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2025 Angsa Robotics
```

Comment:
There's alot of weird spacing issues in this file (too many at times, none at others, etc). Please review. Examples
L27, L63, L81, L91L100, ...